shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -416,9 +416,20 @@
   CGFloat screenScale = [UIScreen mainScreen].scale;
   clippingView.frame = CGRectMake(rect.x() / screenScale, rect.y() / screenScale,
                                   rect.width() / screenScale, rect.height() / screenScale);
+  if (params.has_backdropfilter()) {
```

Comment:
I think when a backdrop filter is applied to a platform view, what we need to do is not to blur the platform view, rather the blur everything else behind the platform view. So I don't think this fix will solve the issue. 

The real issue is that when there is a backdrop filter that is applied to a widget on top of platform view, the platform view has to be blurred.

Reply:
As I know, the blur in skia couldn't have an effect on the UIKit view.
So that I think if we need to blur the UIKit View, we may use the iOS `UIBlurEffect` to blur the UIKit view.

Reply:
Yes, we need to do some trick to make UIKitView blur.

The problem is that if the backdrop filter layer is a parent layer of PlatformView layer, it doesn't mean we need to blur the PlatformView. Instead, we need to blur everything underneath the PlatformView. This is already working.

What the https://github.com/flutter/flutter/issues/43902 is about, IIRC, is when you apply backdrop filter on a widget that is above the PlatformView, the PlatformView needs to be blurred as all the other sibling widgets.

Reply:
I haven't looked into the code, but the `UIBlurEffect` might be the right thing to do. 

We don't want to apply `UIBlurEffect` when the `backdrop filter layer` is the parent, but when a `backdrop filter layer` appears later than the PlatformView during the preorder tree traversal.